src/api/objects/ObjectAPI.js

```diff
@@ -196,20 +196,22 @@ export default class ObjectAPI {
      * @returns {Promise} a promise which will resolve when the domain object
      *          has been saved, or be rejected if it cannot be saved
      */
-    get(identifier, abortSignal) {
+    get(identifier, abortSignal = undefined, forceRemote = false) {
```

Comment:
Would it be more readable to have this be an argument called `useCache` that is `true` by default?

Reply:
This is a little tricky. When this flag is passed in, we skip the cache AND we skip returning any dirty objects from a transaction. If we pass in true, it will automatically resort to fetching the remote object. That was my thinking. That being said, we don't plan to have this around for too long as we'd like to remove the need for transactions altogether. More than happy to discuss more though.